islandora_document.install

```diff
@@ -22,3 +22,14 @@ function islandora_document_uninstall() {
   module_load_include('inc', 'islandora', 'includes/solution_packs');
   islandora_install_solution_pack('islandora_document', 'uninstall');
 }
+
+/**
+ * Update islandora document viewers selection.
```

Comment:
This should be more verbose, fully explaining the consequences of the update.  Islandora should be capitalized.